src/main/java/org/beanpod/switchboard/util/MaintainDeviceStatus.java

```diff
@@ -0,0 +1,93 @@
+package org.beanpod.switchboard.util;
+
+import java.util.Date;
+import java.util.List;
+import java.util.TimeZone;
+import lombok.RequiredArgsConstructor;
+import org.beanpod.switchboard.dao.DeviceDaoImpl;
+import org.beanpod.switchboard.dto.DecoderDto;
+import org.beanpod.switchboard.dto.EncoderDto;
+import org.beanpod.switchboard.dto.StreamDto;
+import org.beanpod.switchboard.dto.mapper.DeviceMapper;
+import org.beanpod.switchboard.entity.DummyInterface;
+import org.beanpod.switchboard.service.LogService;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class MaintainDeviceStatus {
+
+  private static final DateUtil date = new DateUtil();
+  private static final String ONLINE_STATUS = "online";
+  private static final String OFFLINE_STATUS = "offline";
+  private final DeviceDaoImpl service;
```

Comment:
lets change this to deviceDao instead of service because now we have a sperate service package

Reply:
I've updated it